cmd/lncli/commands.go

```diff
@@ -3240,17 +3240,20 @@ var updateChannelPolicyCommand = cli.Command{
 			Usage: "the base fee in milli-satoshis that will " +
 				"be charged for each forwarded HTLC, regardless " +
 				"of payment size",
+			Value: 1,
 		},
-		cli.StringFlag{
+		cli.Float64Flag{
 			Name: "fee_rate",
 			Usage: "the fee rate that will be charged " +
 				"proportionally based on the value of each " +
 				"forwarded HTLC, the lowest possible rate is 0.000001",
+			Value: 0.000001,
```

Comment:
Would rather have these set as a constant in the `routing` package. Otherwise, we'll need to always change this value in _multiple_ places.

Reply:
Also duplicate of #2706. We'll likely merge which ever one implements the feedback first.

Reply:
I find `applyChannelUpdate(...)` in `router.go` but the actual default values are not used anywhere in the code. So where would those "multiple places" be?

Reply:
I found some default values at the top of `chainregistry.go`. But I'm not sure those can be used in other packages since they are located in a `main` package :/

Reply:
@Roasbeef I moved the values to the `lncfg` package and replaced all occurences to use the new defaults.